apps/system/js/app_window.js

```diff
@@ -818,7 +818,9 @@
           // Some signals that chrome needs to respond to can occur before
           // chrome has loaded - in those cases, manually call the handlers.
           if (that.inError) {
-            that.appChrome.handleEvent({type: 'mozbrowsererror'});
+            that.appChrome.handleEvent({type: 'mozbrowsererror', detail: {
+              type: 'non-fatal'
```

Comment:
I don't think we need to add this 'non-fatal' detail, as that type doesn't correspond to any kind of error message that the mozbrowsererror signal would produce. Either we should save the type when we receive the error or we should leave this empty like before, in my opinion.